.github/portfolio.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+</head>
+<body>
+    <div class="section">
+    <h2>Here are some projects that I've worked on</h2>coid
```

Comment:
make sure that child elements are consistently indented underneath the parent elements! this is demonstrated well across the project, but take another closer look for instances such as these where the child element should be indented.